src/ensembl/src/global/globalReducer.ts

```diff
@@ -31,6 +31,11 @@ export default function globalReducer(
         ...state,
         scrollPosition: { ...state.scrollPosition, ...action.payload }
       };
+    case getType(global.toggleCommunicationPanel):
+      return {
+        ...state,
+        isCommunicationPanelOpen: !state.isCommunicationPanelOpen
```

Comment:
Umm, let's create a dedicated slice for the communication. I have a feeling lots of communications-specific concerns are coming our way...

Reply:
That's done